resources/assets/js/components/JobDetails/JobDetails.tsx

```diff
@@ -891,20 +905,55 @@ const JobDetails: React.FunctionComponent<
                     },
                   )}
                 </RadioGroup>
-                <div data-c-alignment="centre" data-c-grid-item="base(1of1)">
-                  <button
-                    data-c-button="solid(c1)"
-                    data-c-dialog-action="open"
-                    data-c-radius="rounded"
-                    type="submit"
-                    disabled={isSubmitting}
+                <div data-c-grid="gutter" data-c-grid-item="base(1of1)">
+                  <div data-c-grid-item="base(1of1)">
+                    <hr data-c-margin="top(normal) bottom(normal)" />
+                  </div>
+                  <div
+                    data-c-alignment="base(centre) tp(left)"
+                    data-c-grid-item="tp(1of2)"
                   >
-                    <FormattedMessage
-                      id="jobDetails.submitButtonLabel"
-                      defaultMessage="Next"
-                      description="The text displayed on the submit button for the Job Details form."
-                    />
-                  </button>
+                    <button
+                      data-c-button="outline(c2)"
+                      data-c-radius="rounded"
+                      type="button"
+                      disabled={isSubmitting}
+                      onClick={(): void => {
+                        /** TODO:
+                         * This is a race condition, since the setState hook call is asynchronous.
+                         * I have to find a way to handle 2 submit buttons in formik without a race condition somewhere :(
+                         * For now, the setState always happens faster than the validation check, so it works.
+                         * See https://github.com/jaredpalmer/formik/issues/214
+                         * -- Tristan
+                         */
+                        setReturnOnSubmit(true);
```

Comment:
This doesn't seem to be working for me locally. On click, nothing happens, and then if I click the `Preview` button it will navigate back to the intro page :(.

Only seems to happen on the Details page for me for some reason.

Reply:
Try again now. I think an issue with the Job api call (classification_code not returning correctly) was messing it up.